documentation/modules/security/ref-certificates-and-secrets.adoc

```diff
@@ -78,7 +78,7 @@ Only the `_<cluster_name>_-cluster-ca-cert` secret is required by clients.
 All other cluster secrets are accessed by Strimzi components.
 You can enforce this using Kubernetes role-based access controls, if necessary.
 
-NOTE: The CA certificates in `_<cluster_name>_-cluster-ca-cert` must be trusted by Kafka client applications so that they validate the Kafka broker certificates when connecting to Kafka brokers over TLS.
+NOTE: The CA certificates in `_<cluster_name>_-cluster-ca-cert` must be trusted by Kafka client applications so that they validate the Kafka broker certificates when connecting to Kafka brokers over mTLS.
```

Comment:
```suggestion
NOTE: The CA certificates in `_<cluster_name>_-cluster-ca-cert` must be trusted by Kafka client applications so that they validate the Kafka broker certificates when connecting to Kafka brokers over TLS.
```